internal/statefulsyncer/stateful_syncer.go

```diff
@@ -155,3 +157,60 @@ func (s *StatefulSyncer) BlockRemoved(
 
 	return err
 }
+
+// EndAtTipLoop runs a loop that evaluates end condition EndAtTip
+func (s *StatefulSyncer) EndAtTipLoop(
+	ctx context.Context,
+	tipDelay int64,
+	interval time.Duration,
```

Comment:
Why do we have this as an arg? This seems like it should be a const somewhere?

Reply:
It is a constant used by caller https://github.com/coinbase/rosetta-cli/blob/luke/enhance-end-condition/internal/tester/data.go#L58-L62